README.md

```diff
@@ -18,6 +18,23 @@ libraryDependencies ++= Seq(
 )
 ```
 
+## Why log4cats?
+
```

Comment:
I'd also include another sentence of the sort:
```
Logging is oftimes an overlooked side-effect, which under the hood can range from writing to a mutable queue, writing to disk, outputing to the console, or sometimes even doing network I/O! To correctly deal with these kinds of side-effects we have to ensure they are properly wrapped in `IO`, see the [cats-effect docs](https://typelevel.org/cats-effect/docs/concepts#side-effects) for more details.
```

And then tie it in with the sentence you wrote.

Now I realize this essentially doubles the amounts of words :sweat_smile: it definitely requires _some_ editing :D, and I'm not clinging to any particular form or formulation. Just want to point readers to the fact that there is documentation around dealing with side-effects.

Reply:
I see your point.  The purpose of this example was targeted towards new learners and people possibly in a hybrid state that want to be using cats-effect with their existing app.  The goal was to give something easy to understand, but also valid with the cats-effect ethos.  So I don't think I should include it in the "Why log4cats?" section...but I will add it further down

Reply:
Pushed change.  Let me know what you think?

Reply:
Nice!